google-cloud-firestore/lib/google/cloud/firestore/document_listener.rb

```diff
@@ -119,6 +126,83 @@ def stop
         def stopped?
           @listener.stopped?
         end
+
+        ##
+        # Register to be notified of errors when raised.
+        #
+        # If an unhandled error has occurred the listener will attempt to
+        # recover from the error and resume listening.
+        #
+        # Multiple error handlers can be added.
+        #
+        # @yield [callback] The block to be called when an error is raised.
+        # @yieldparam [Exception] error The error raised.
+        #
+        # @example
+        #   require "google/cloud/firestore"
+        #
+        #   firestore = Google::Cloud::Firestore.new
+        #
+        #   # Get a document reference
+        #   nyc_ref = firestore.doc "cities/NYC"
+        #
+        #   listener = nyc_ref.listen do |snapshot|
+        #     puts "The population of #{snapshot[:name]} "
+        #     puts "is #{snapshot[:population]}."
```

Comment:
Maybe put these in one line. Same with other examples below. If it's too long to fit on one line, then you don't need the trailing space on the first line.

Reply:
Fixed.